templates/single_album_page.html

```diff
@@ -11,7 +11,7 @@ <h4>Description</h4>
         <ul>
             <li>published by: {{ query.publisher }}</li>
             <li>release on: {{ query.release_date }}</li>
-            <li> listen on spotify at: <a href="{{ query.spotify_url }}"
+            <li> listen on spotify at: <a href="{{ query.spotify_url }}"></a>
```

Comment:
Thanks for catching another one of my typos! (I do this kind of thing all the time...it's bad...)